src/java/org/apache/cassandra/cql3/constraints/ScalarColumnConstraint.java

```diff
@@ -80,8 +80,8 @@ public void evaluate(AbstractType<?> valueType, ByteBuffer columnValue)
         }
 
         if (!relationType.isSatisfiedBy(valueType, columnValue, value))
-            throw new ConstraintViolationException("Column value does not satisfy value constraint. "
-                                                   + " It should be " + relationType + " " + term);
+            throw new ConstraintViolationException("Column value does not satisfy value constraint for column " + param + ". "
```

Comment:
@bbotella could you please put "param" into single quotes? 

`for column '" + param + "'."`